lib/extract-js.js

```diff
@@ -0,0 +1,216 @@
+var htmlparser = require('htmlparser2');
+var Errors = require('./errors');
+
+/**
+ * Html file representation (needs for errors output).
+ *
+ * @name HtmlFile
+ */
+var HtmlFile = function(filename, source) {
+    this._filename = filename;
+    this._lines = source.split(/\r\n|\r|\n/);
+};
+
+HtmlFile.prototype = {
+    /**
+     * Returns source filename for this object representation.
+     *
+     * @returns {String}
+     */
+    getFilename: function() {
```

Comment:
You mean `HtmlFile.prototype.getFilename = function() { .. }` should be here?
I wrote this code as close as possible to [JsFile style](https://github.com/jscs-dev/node-jscs/blob/master/lib/js-file.js#L60).